contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,82 @@
+Hence, Validation Suites are not subject to the deprecation cycles such as those 
+that apply to features and modification decisions are left to the decision making
+provess(es) of the SIG.  The end users of the Validation Suites are strongly 
+encouraged to participate in the upkeep and maintenance of the suite so
+they know what changes are coming up and can adapt their 
+product/implementation to the updates of the test suite.
+
+## Running Validation Suite Tests
+
+Validation Suite tests are designed to be run even when there is no cloud provider
+configured. The [Testing Commons](https://github.com/kubernetes/community/tree/master/sig-testing) 
+subproject within the Kubernetes sig-testing
+community is currently tasked with adding a framework for writing test suites and
+this new framework may be used for running Validation Suites. This new framework is 
+optional because there is not a prescribed approach for how a Validation Suite needs to 
+be run.  Instead, the authors of the Validation Suite need to ensure that the 
+mechanisms for running the Validation Suite are easy to use, provide sufficient
+documentation, and follow patterns similar to existing e2e tests. 
+In addition, the mechanism should provide the  ability to select some tests and 
+skip others.  
+
+### Validation Suite Test Comment Metadata
+
+Each Validation suite test must include the following piece of metadata
+within its associated comment:
+
+- `Release`: indicates the Kubernetes release that the test was added to the
+  Validation test suite. If the test was modified in subsequent releases
+  then those releases should be included as well (comma separated)
+- `Testname`: a human readable short name of the test
+- `Description`: a detailed description of the test. This field must describe
+  the required behaviour of the Kubernetes components being tested using
+  [RFC2119](https://tools.ietf.org/html/rfc2119) keywords. This field
+  is meant to be a "specification" of the tested Kubernetes features, as
+  such, it must be detailed enough so that readers can fully understand
+  the aspects of Kubernetes that are being tested without having to read
+  the test's code directly. Additionally, this test should provide a clear
+  distinction between the parts of the test that are there for the purpose
+  of validating Kubernetes rather than simply infrastructure logic that
+  is necessary to setup, or clean up, the test.
+
+### Sample Validation Suite Test
+
+The following snippet of code shows a sample Storage Validation Suite test's 
+metadata.  Note how the test contains a bracketed label with name of the Validation
+Suite that this test belongs to. 
+
+```
+/*
+  Release : v1.14
+  Testname: CSI Attach, Non-attachable volume used in a Pod
+  Description: When CSI driver attach is called on non attachable volumes it should not create a volume attachment.
+*/
+It("should not require VolumeAttach for drivers without attachment [StorageValidation]", func() {
```

Comment:
nit: This does not seem to correspond with the description or name given above, so again, the example is quite confusing.